isnts/templates/donors/login.html

```diff
@@ -2,24 +2,202 @@
 {% load staticfiles %}
 {% block title %}Login{% endblock %}
 {% block content %}
-<section class="row">
-    <div class="col s6">
-    <h2>Login</h2>
-        <form method="post" role="form">
-            {% for field in login_form.visible_fields %}
-            <div id="div_{{ field.html_name }}" class="form-group{% if field.errors %} has-error{% endif %} input-field">
-                {{ field }} {{ field.label_tag }}
-                <div class="help-block">
-                    {{ field.errors }}
+
+<script>
+$(document).ready(function(){
+    $('.parallax').parallax();
+});
+
+jQuery(document).ready(function($){
```

Comment:
Preco deklarujes tu istu funkciu dva krat pod sebou? Nie je lepsie mat to v jednej?